x-pack/plugins/alerting/server/routes/create_rule.ts

```diff
@@ -35,14 +37,7 @@ export const bodySchema = schema.object({
   schedule: schema.object({
     interval: schema.string({ validate: validateDurationSchema }),
   }),
-  actions: schema.arrayOf(
-    schema.object({
-      group: schema.string(),
-      id: schema.string(),
-      params: schema.recordOf(schema.string(), schema.any(), { defaultValue: {} }),
-    }),
-    { defaultValue: [] }
-  ),
+  actions: actionsSchema,
   notify_when: schema.string({ validate: validateNotifyWhenType }),
```

Comment:
we might need to use `schema.maybe` because we do not need to set it up and we should do the same with `throttle`